tests/lora/test_lora_layers_cogvideox.py

```diff
@@ -126,6 +127,7 @@ def get_dummy_inputs(self, with_generator=True):
         return noise, input_ids, pipeline_inputs
 
     @skip_mps
+    @pytest.mark.xfail("Test currently fails on CPU and PyTorch 2.5.1 but not on PyTorch 2.4.1.", strict=True)
```

Comment:
I think you need to add the [`condition`](https://docs.pytest.org/en/stable/reference/reference.html#pytest-mark-xfail-ref) that the device is CPU, otherwise the test will fail on GPU, since it's `strict`.